### PR TITLE
feat(action): Add COE docker resource management action(s)

### PR DESCRIPTION
**Pull request by ignacio-penas**

### CODE REVIEW

File: nuvla/job_engine/connector/nuvlaedge_docker.py [hunk 1]
```diff
@@ -763,3 +770,163 @@ def log(self, component: str, since: datetime, lines: int) -> str:
...
+            'message': f"Unsupported action: {action}"
+        }
+
+    @staticmethod
+    def _unsupported_resource_for_action_job_response(resource: str, action: str):
+        return {
+            'success': False,
+            'return-code': 0,
+            'message': f"Unsupported resource {resource} for action {action}"
+        }
```
Comment (schaubl) [inline nuvla/job_engine/connector/nuvlaedge_docker.py:788]: You could use the method `_new_job_response` below _(if you convert these 2 static methods to class methods_).

Comment (ignacio-penas) [reply nuvla/job_engine/connector/nuvlaedge_docker.py:788]: Agree, `_new_job_response` method was added after these to and overlooked it :)

**Comment by schaubl** [comment]
@ignacio-penas can you extract the new methods of _nuvlaedge_docker.py_ to a new dedicated subclass of `NuvlaBox` ?
And also move it in a new file to prevent the current file to grow even more.
Maybe you can put this new file in a new subfolder (something like `nuvlaedge/coe_resources_docker.py` or `nuvlaedge_docker/coe_resources.py`).

**Comment by ignacio-penas** [comment]
> @ignacio-penas can you extract the new methods of _nuvlaedge_docker.py_ to a new dedicated subclass of `NuvlaBox` ? And also move it in a new file to prevent the current file to grow even more. Maybe you can put this new file in a new subfolder (something like `nuvlaedge/coe_resources_docker.py` or `nuvlaedge_docker/coe_resources.py`).

Done. I have created a single file coe_resource.py which could contain both docker and k8s resource handlers

**Comment by schaubl** [comment]
@ignacio-penas could you check if we are in pull mode _(I think there is a function for that in Job)_ and fail if not the case ?
Also can you add an error message in the output if actions for an unsupoorted coe (anything else than docker) are provided ?

**Pull request merged**